frontend/src/components/SeedlotRegistrationSteps/OwnershipStep/utils.ts

```diff
@@ -163,7 +162,7 @@ export const skipForInvalidLength = (name: string, value: string): boolean => {
   return false;
 };
 
-const isInputEmpty = (value: string | number | null) => {
+const isInputEmpty = (value: string | number | DropDownObj | null) => {
   // null can be the value even with the type check
   if (value === '' || value === null) {
```

Comment:
I know this code was not changed, but I wonder if we could have something like `return !value;` instead of this if?